### ISSUE TITLE
add_project() should return the project object instead of True

### ISSUE THREAD

**Issue by alvaroabascar** [mint-labs/mintlabs#14]

**Comment by goals2008** [comment]
Need to have an id returned to instantiate the project in next step in docs
# Instantiate the project using it's ID

**Comment by alvaroabascar** [comment]
I have just released version 0.1.5. Now account.add_project() returns an instance of Project(), ready to use:

```{python}
import mintlabs
account = mintlabs.Account('user', 'password')
project = account.add_project('abbreviation', 'name')

# project is an instance of Project. We already can, for example, add a new subject
subject = mintlabs.Subject('subject_name')
project.add_subject(subject)
```
